lib/rails_multisite/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 #
 module RailsMultisite
-  VERSION = "5.0.0"
+  VERSION = "5.0.1"
```

Comment:
@Flink @ZogStriP Shouldn't this be a major version bump since dropping support for a Rails version is backwards incompatible?

Reply:
If I’m not mistaken, the gem should still work with Rails 6.0, but as we’re not using it anymore anywhere, I dropped it from the test suite. (so not supported officially).
We can do another bump if you prefer though :sweat_smile:

Reply:
O I noticed that Ruby 3.0 is EOL at the end of this month so I dropped support for it in https://github.com/discourse/rails_multisite/pull/33 and did a major version bump.